integration/helpers/base_test.py

```diff
@@ -136,15 +139,29 @@ def create_and_verify_stack(self, file_name, parameters=None):
         parameters : list
             List of parameters
         """
+        folder, file_name = file_name.split("/")
         self.output_file_path = str(Path(self.output_dir, "cfn_" + file_name + ".yaml"))
-        self.expected_resource_path = str(Path(self.expected_dir, file_name + ".json"))
+        self.expected_resource_path = str(Path(self.expected_dir, folder, file_name + ".json"))
         self.stack_name = STACK_NAME_PREFIX + file_name.replace("_", "-") + "-" + generate_suffix()
 
-        self._fill_template(file_name)
+        self._fill_template(folder, file_name)
         self.transform_template()
         self.deploy_stack(parameters)
         self.verify_stack()
 
+    def update_stack(self, file_name, parameters=None):
+        if os.path.exists(self.output_file_path):
+            os.remove(self.output_file_path)
+        if os.path.exists(self.sub_input_file_path):
+            os.remove(self.sub_input_file_path)
+
+        folder, file_name = file_name.split("/")
```

Comment:
Because the file_name is in special format ("subfolder/name"), it would be better to add docstring about it.

Reply:
Rename the `file_name` param to `file_path` maybe.
Same for `create_and_verify_stack` above.
Add docstrings indeed.

Reply:
change `file_name` to `file_path`, added doc string as well